R/sulpher.R

```diff
@@ -71,11 +71,11 @@ calc_slv <- function(B_LU_BRP, B_SOILTYPE_AGR, B_AER_CBS,A_SOM_LOI,A_S_RT, D_BDS
   
   # Calculate SLV for grass (deze formule: Stot = g/kg en dichtheid in g/cm3)
   dt.grass <- dt[crop_category == "grasland"]
-  dt.grass[, value := 17.8 * A_S_RT * 0.001 * D_BDS * 0.001]
+  dt.grass[, value := 17.8 * A_S_RT * D_BDS * 0.001]
   
```

Comment:
sorry, this is incorrect.  see page 18 in Bemestingsadvies grasland. Soils are low in S supply when S supplying capacity is lower than 17 and its high when SLV exceeds the 23 (for grassland). 
If S total is 650 mg/kg, then SLV is around 15 kg / ha. This change would result in a S supply of 15.000 kg S/ha, that seems unrealistic to me.